contrib/storage-splunk/README.md

```diff
@@ -42,6 +42,10 @@ Sometimes Splunk has issue in connection to it:
 https://github.com/splunk/splunk-sdk-java/issues/62 <br>
 To bypass it by Drill please specify "reconnectRetries": 3. It allows you to retry the connection several times.
 
+### User Translation
+The Splunk plugin supports user translation.  Simply set the `authMode` parameter to `USER_TRANSLATION` and use either the plain or vault credential provider for credentials.
```

Comment:
Do you think I should make authMode values case insensitive before 1.21 is released? USER_TRANSLATION looks odd compared to other Drill config values and is only in caps due to the coincidence that the Java Enum names are in caps.

Reply:
That's probably a good idea.  I'd give it a +1.